docs/notebooks/recovering_from_errors.pct.py

```diff
@@ -41,14 +41,13 @@ def __call__(self, x):
 
 # %% [markdown]
 # ## Set up the problem
-# We'll use the same set up as before, except for the acquisition rule, where we'll use `TrustRegion`, which (with non-trivial state) will better illustrate how to recover.
+# We'll use the same set up as before, except we'll also use a `TrustRegion`, which (with non-trivial state) will better illustrate how to recover.
```

Comment:
Maybe elaborate what non-trivial state means here? Not obvious to those not ordained in trust regions/Trieste

Reply:
done. is that what you had in mind?